packages/core/dao/VotingManagerDAO.js

```diff
@@ -4,8 +4,8 @@
  */
 
 import BigNumber from 'bignumber.js'
-import Immutable from 'immutable'
-import ipfs from '../utils/IPFS'
+import { List } from 'immutable'
+import ipfs from '@chronobank/core/utils/IPFS'
```

Comment:
This is wrong merge. It must be `import ipfs from '../utils/IPFS'`
So, rebase results are compromised.